RecoTracker/IterativeTracking/python/SiStripTripletStep_cff.py

```diff
@@ -0,0 +1,305 @@
+    ComponentName = cms.string('siStripTripletStepRKFitter'),
+    minHits = 8
+)
+
+
+siStripTripletStepRKTrajectoryFitterForLoopers = siStripTripletStepRKTrajectoryFitter.clone(
+    ComponentName = cms.string('siStripTripletStepRKFitterForLoopers'),
+    Propagator = cms.string('PropagatorWithMaterialForLoopers'),
+)
+
+siStripTripletStepRKTrajectorySmoother = TrackingTools.TrackFitters.RungeKuttaFitters_cff.RKTrajectorySmoother.clone(
+    ComponentName = cms.string('siStripTripletStepRKSmoother'),
+    errorRescaling = 10.0,
+    minHits = 8
+)
+
+
+siStripTripletStepRKTrajectorySmootherForLoopers = siStripTripletStepRKTrajectorySmoother.clone(
+    ComponentName = cms.string('siStripTripletStepRKSmootherForLoopers'),
+    Propagator = cms.string('PropagatorWithMaterialForLoopers'),
+)
+
+import TrackingTools.TrackFitters.FlexibleKFFittingSmoother_cfi
+siStripTripletFlexibleKFFittingSmoother = TrackingTools.TrackFitters.FlexibleKFFittingSmoother_cfi.FlexibleKFFittingSmoother.clone(
+    ComponentName = cms.string('siStripTripletFlexibleKFFittingSmoother'),
+    standardFitter = cms.string('siStripTripletStepFitterSmoother'),
+    looperFitter = cms.string('siStripTripletStepFitterSmootherForLoopers'),
+)
+
+
+#----------------------------------------- TRACK FITTING
+#import RecoTracker.TrackProducer.TrackProducer_cfi
+#siStripTripletStepTracks = RecoTracker.TrackProducer.TrackProducer_cfi.TrackProducer.clone(
+#    src = 'siStripTripletStepTrackCandidates',
+#    AlgorithmName = cms.string('siStripTripletStep'),
+#    Fitter = cms.string('FlexibleKFFittingSmoother')
+#    )
+#
+#from TrackingTools.TrajectoryCleaning.TrajectoryCleanerBySharedHits_cfi import trajectoryCleanerBySharedHits
+#siStripTripletStepTrajectoryCleanerBySharedHits = trajectoryCleanerBySharedHits.clone(
+#        ComponentName = cms.string('siStripTripletStepTrajectoryCleanerBySharedHits'),
+#            fractionShared = cms.double(0.16),
+#            allowSharedFirstHit = cms.bool(True)
+#            )
+#siStripTripletStepTrackCandidates.TrajectoryCleaner = 'siStripTripletStepTrajectoryCleanerBySharedHits'
+
+import RecoTracker.TrackProducer.TrackProducer_cfi
+siStripTripletStepTracks = RecoTracker.TrackProducer.TrackProducer_cfi.TrackProducer.clone(
+    src = 'siStripTripletStepTrackCandidates',
+    AlgorithmName = cms.string('siStripTripletStep'),
+    #Fitter = 'siStripTripletStepFitterSmoother',
+    Fitter = 'siStripTripletFlexibleKFFittingSmoother',
+    )
+
+
+#---------------------------------------- TRACK SELECTION AND QUALITY FLAG SETTING.
+
+from RecoTracker.FinalTrackSelectors.TrackMVAClassifierPrompt_cfi import *
+from RecoTracker.FinalTrackSelectors.TrackMVAClassifierDetached_cfi import *
+siStripTripletStepClassifier1 = TrackMVAClassifierDetached.clone()
+siStripTripletStepClassifier1.src = 'siStripTripletStepTracks'
+siStripTripletStepClassifier1.mva.GBRForestLabel = 'MVASelectorIter6_13TeV'
```

Comment:
uhmmmm, are you sure you wnat to make use of a BDT output which has been trained for the tobTec step in a different phase space ?